packages/cli/src/commands/generate/scaffold/scaffold.js

```diff
@@ -8,6 +8,7 @@ import { Listr } from 'listr2'
 import { paramCase } from 'param-case'
 import pascalcase from 'pascalcase'
 import terminalLink from 'terminal-link'
+import { prettify } from '../../../lib/index'
```

Comment:
Tip: the imports from `'../../../lib'` (lines 17-27) are actually imports from `'../../../lib/index'`, so you could move this prettify import to that group below.

Reply:
ah! Thank you for pointing it out!